custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -317,11 +318,23 @@ def transform(self, model=None, **kwargs):
                 input_binary_data,
                 kwargs.get(StructuredDtoKeys.MIMETYPE),
             )
+            if target_binary_data is not None:
+                target_data = StructuredInputReadUtils.read_structured_input_data_as_df(
+                    target_binary_data, kwargs.get(StructuredDtoKeys.TARGET_MIMETYPE)
+                )
+            else:
+                target_data = None
```

Comment:
can you colocate this with the function call below inside that if statement?

Reply:
I may break the `transform` method out into `preprocess`, which just gets called for non-transform cases before passing on to predict, and `transform`, which gets called only for the custom transform flow